sklego/meta/hierarchical_predictor.py

```diff
@@ -318,11 +321,14 @@ def _predict_estimators(self, X, method_name):
             X = nw.from_native(pd.DataFrame(X))
 
         n_samples = X.shape[0]
-        native_space = nw.get_native_namespace(X)
+        native_namespace = nw.get_native_namespace(X)
+        global_series = nw.from_dict({self._GLOBAL_NAME: [1] * n_samples}, native_namespace=native_namespace)[
```

Comment:
Before putting it in narwhals we make a big list of ones which we put into a dictionary before it goes into narwhals. I wonder ... is there a faster way? My gut feeling is that `np.ones()` is a better idea here but I will gladly hear it if I am missing something.

Reply:
Checking if all tests will pass